### PR TITLE
Fix GradScaler + Perceptual Loss

### PR DESCRIPTION
**Pull request by terrainer**
Reduced the init_scale of GradScaler to 2**10,  by lowering the scaling value, GradScaler no longer produces invalid values and ensures that optimizer.step() is executed correctly. This in turn resolves the warning for calling lr_scheduler.step() before optimizer.step().

### CODE REVIEW

**Comment by muslll** [comment]
[Commited](https://github.com/muslll/neosr/commit/f342317c2186ac5dd0ffdfd098a88ae7bda35e33). Sorry for not merging your PR directly, I've mentioned you in the commit, as well as on the [issue](https://github.com/muslll/neosr/issues/4) and on [readme](https://github.com/muslll/neosr#license-and-acknowledgements). Thanks.
